standalone/test/unit_tests/data_conversion_layer/unittest_laserscan_conversions.cpp

```diff
@@ -307,13 +303,12 @@ TEST(LaserScanConversionTest, conversionShouldIgnoreEmptyFramesForMonitoringFram
   // The following from_theta's are a real example from wireshark.
   // (angle_start:=-0.1, angle_end:=0.1)
   std::vector<Stamped> stamped_msgs = {
-    Stamped(MessageBuilder().fromTheta(Tenth(2500)).resolution(Tenth(2)).scanCounter(42).measurements({}), 3),
-    Stamped(MessageBuilder().fromTheta(Tenth(0)).resolution(Tenth(2)).scanCounter(42).measurements({}), 4),
-    Stamped(MessageBuilder().fromTheta(Tenth(500)).resolution(Tenth(2)).scanCounter(42).measurements({}), 5),
-    Stamped(MessageBuilder().fromTheta(Tenth(1318)).resolution(Tenth(2)).scanCounter(42).measurements({ 1., 2., 3. }),
-            6),
-    Stamped(MessageBuilder().fromTheta(Tenth(1500)).resolution(Tenth(2)).scanCounter(42).measurements({}), 7),
-    Stamped(MessageBuilder().fromTheta(Tenth(2000)).resolution(Tenth(2)).scanCounter(42).measurements({}), 8)
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(2500)).measurements({}), 3),
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(0)).measurements({}), 4),
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(500)).measurements({}), 5),
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(1318)).measurements({ 1., 2., 3. }), 6),
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(1500)).measurements({}), 7),
+    Stamped(createDefaultMsgBuilder().fromTheta(Tenth(2000)).measurements({}), 8)
   };
 
   ASSERT_NO_THROW(data_conversion_layer::LaserScanConverter::toLaserScan(stamped_msgs));
```

Comment:
I knwo you didn't change this but what exactly does this test ... test?
"conversionShouldIgnoreEmptyFramesForMonitoringFramesValidation"
... some frames with empty monitoring frames and different theta
"ASSERT_NO_THROW(data_conversion_layer::LaserScanConverter::toLaserScan(stamped_msgs));"

what would be different if they where not empty?

Reply:
`conversionShouldIgnoreEmptyFramesWhenCheckingIfFromThetasFitTogether` - better?